### PR TITLE
Additive import

### PR DESCRIPTION
**Pull request by BryonLewis**
This is a PR that was an upstream branch for some other deployments.  For right now these changes are in the Web Version only.  I can make a separate PR to incorporate these changes into the electron version.

Creates an additive import option for `postprocess`:
`additive` means that if an annotation file is found during postprocess it will add the tracks to existing tracks instead of doing a replacement.
`additive_prepend` is a value that is prepended to all types of the new tracks found.  This is to easily filter the tracks in the type editor between the new and old tracks.

Code Changes:

- Post process has additional flags 'additive' to indicate that the import should be additive to the current tracks. 
- 'additivePrepend' is a prepended type that should be added to all tracks imported using additive.  I.E, if you have existing tracks named fish and the import has more tracks named fish, if you use an additivePrepend of 'newer' then all of the new annotations will be renamed to 'newer_fish' for their type.
- There are some UI changes to support this additive prepend functionality.  Mostly allowing from the interface to check off an import as being additive and providing a Prepend name value.
- Fix for transcoding so that videos that contain multiple video streams it will use the first videostream
- Custom sorting for annotations.  Annotations in CSV files were occasionally given in the wrong frame order.  This attempts to fix that order by custom sorting the CSV rows in python.

I apologize for the commit history, if required I can squash it down.  During the merge it will become squashed anyways.

### CODE REVIEW

File: client/dive-common/components/ImportAnnotations.vue [hunk 1]
```diff
@@ -133,16 +149,33 @@ export default defineComponent({
...
+                label="Additive"
+              />
+            </v-row>
+            <div v-if="additive">
+              <v-text-field
+                v-model="additivePrepend"
+                label="Prepend to types"
+                clearable
+              />
+            </div>
```
Comment (marySalvi) [inline client/dive-common/components/ImportAnnotations.vue:176]: Perhaps a tooltip or hint to explain the functionality of this prepend text

File: client/dive-common/components/ImportAnnotations.vue [hunk 2]
```diff
@@ -133,16 +149,33 @@ export default defineComponent({
...
+                @click="openUpload"
+              >
+                Import
+              </v-btn>
+            </v-row>
+            <v-row>
+              <v-switch
+                v-model="additive"
+                label="Additive"
+              />
```
Comment (marySalvi) [inline client/dive-common/components/ImportAnnotations.vue:168]: I feel like `additive` isn't intuitive. Perhaps instead a checkbox with an 'overwrite' label defaulted to checked.

File: client/platform/desktop/backend/native/common.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import mime from 'mime-types';
 import moment from 'moment';
 import lockfile from 'proper-lockfile';
 import {
-  cloneDeep, merge, uniq, pick,
+  cloneDeep, merge, uniq, pick, add, max,
```
Comment (marySalvi) [inline client/platform/desktop/backend/native/common.ts:12]: looks like `add` and `max` isn't used

**Review by marySalvi — CHANGES_REQUESTED** [review]

**Comment by BryonLewis** [comment]
I am completely open to any other wording or suggestions that you may have for it.  I also looked through the code and it looks like I already implemented the Desktop version of this (It's been a branch for too long and I forgot :facepalm: )

**Comment by marySalvi** [comment]
> I also looked through the code and it looks like I already implemented the Desktop version of this (It's been a branch for too long and I forgot  )

Heh I noticed that but I thought maybe it wasn't fully hooked up yet. I'll do a visual check on desktop, already checked the code.

**Review by marySalvi — APPROVED** [review]

**Pull request merged**